### PR TITLE
Create american-journal-of-science.csl

### PR DESCRIPTION
**Pull request by damnation333**
As requested in March 2016 with US Geological Journal as basis:
https://forums.zotero.org/discussion/57283/style-request-american-journal-of-science/

### CODE REVIEW

File: american-journal-of-science.csl [hunk 1]
```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>American Journal of Science</title>
+    <title-short>AJS</title-short>
+    <id>http://www.zotero.org/styles/american-journal-of-science</id>
+    <link href="http://www.zotero.org/styles/american-journal-of-science" rel="self"/>
```
Comment (rmzelle) [inline american-journal-of-science.csl]: @damnation333, please add a "template" link.

File: american-journal-of-science.csl [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    <link href="http://www.nwrc.usgs.gov/techrpt/sta28.pdf" rel="documentation"/>
```
Comment (rmzelle) [inline american-journal-of-science.csl]: Documentation link probably needs to be changed, right?

Comment (rmzelle) [inline american-journal-of-science.csl]: http://www.ajsonline.org/site/misc/ifora.xhtml, presumably.

Comment (damnation333) [inline american-journal-of-science.csl]: sorry. I started all right and then something effed up and I went back to the template and I forgot to change these things. very sorry.

File: american-journal-of-science.csl [hunk 3]
```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>American Journal of Science</title>
+    <title-short>AJS</title-short>
+    <id>http://www.zotero.org/styles/american-journal-of-science</id>
+    <link href="http://www.zotero.org/styles/american-journal-of-science" rel="self"/>
+    <link href="http://www.zotero.org/styles/us-geological-society" rel="template"/>
```
Comment (rmzelle) [inline american-journal-of-science.csl]: @damnation333, should this be "us-geological-survey"?

**Comment by rmzelle** [comment]
Thanks again!

**Pull request merged**